browser/resources/settings/brave_overrides/people_page.ts

```diff
@@ -50,6 +50,14 @@ RegisterPolymerTemplateModifications({
     } else {
       console.error('[Brave Settings Overrides] People Page cannot find syncSetup/advanced template')
     }
+    const syncSetupPageContent = templateContent.querySelector(
```

Comment:
Q) It's also leftover and could be deleted as we have our own sync page?

Reply:
Yes, we don't seem to be using this page. It seems to have something to do with the Privacy Guide, which we disable anyway. When using search in settings, if not removed, this template causes an exception because it can't find the trigger control for the page.